src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -15,8 +15,8 @@
 public class DriveSubsystem extends SubsystemBase{
 
 
-    private final SwerveModules[] driveModules = new SwerveModules[4]; // FR, RL, BR, BL
-    private final PigeonInterface pigeonInterface = new PigeonInterface(); 
+    private final SwerveModules[] driveModules = new SwerveModules[4]; // TODO: This will be the future format: FR, RL, BR, BL
```

Comment:
Isn't this FL, FR, BL, BR now?

Reply:
It was inconsistent throughout the code so I refactored it yesterday. Just wanted to keep this note to talk about it tomorrow.

Reply:
yeah, I just meant that the comment has "RL" which is sort of ambiguous

Reply:
Oops, I meant to correct that before. Sorry